appengine/metadata/flexible/app.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2016, Google, Inc.
```

Comment:
Is this sample supposed to be split into `flexible` and `standard` variants? (If memory serves, that was only the for the quickstarts.)

Reply:
The original sample retrieves the instance IP address, which is not available on std (always returns `localhost` or `None`). Now the flex sample still retrieves IP address but the std sample retrieves project id instead. Should we use the std sample for both?